config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
It would be nicer to avoid the coupling from the beginning, so that if other people extend these features with other scenarios which need the same config, they can immediately reuse it rather than giving in to the temptation to copy'n'paste the config.  But if we both agree to watch out for DRY violations then I guess we can tolerate this coupling for now :)